src/index.tsx

```diff
@@ -98,6 +98,7 @@ const ModalizeBase = (
     tapGestureEnabled = true,
     closeOnOverlayTap = true,
     closeSnapPointStraightEnabled = true,
+    disablePanGestureForChildren = false,
```

Comment:
Would you mind renaming to:

```suggestion
    panGestureChildrenEnabled = true,
```

to follow the same naming convention